jobs/worker/spec

```diff
@@ -156,6 +156,13 @@ properties:
       Driver to use for the volume store. One of detect, overlay, btrfs, or naive.
     default: detect
 
+  baggageclaim.bind_ip:
+    env: CONCOURSE_BAGGAGECLAIM_BIND_IP
+    description: |
+      IP on which Baggageclaim should listen for HTTP traffic.
+      When p2p is enabled this needs to be set to 0.0.0.0
+    example: 0.0.0.0
```

Comment:
Could you add a `default: 127.0.0.1` here just to follow convention like bind_port?

Reply:
i can.
but when you don't use this parameter `bind_ip` it will by default already be `127.0.0.1`
so thats why i thought it would be wise not to set a default at all

Reply:
So is [bind_port](https://github.com/concourse/baggageclaim/blob/4b89831469e786e63d2290b12e60a031dfcf688f/baggageclaimcmd/command.go#L24). However we still explicity call it out to `7788` by default so Bosh OPs knows what value bind port is without needed to check the binary help message or look it up in codebase.